generator/tree_test.go

```diff
@@ -1881,8 +1882,8 @@ func TestGenerateConfigModule(t *testing.T) {
 			}
 		}
 
-		nameToNode := prepareTree(c.node)
-		got := generateConfigModule(c.cfg, c.node, nameToNode)
+		nameToNode := prepareTree(c.node, log.NewNopLogger())
+		got, _ := generateConfigModule(c.cfg, c.node, nameToNode, log.NewNopLogger())
```

Comment:
Are you sure we should ignore the error here? Previously these were `Fatal` exits.

Reply:
The got will be empty, so it'll be caught that way

Reply:
But it means that the Fatal errors printed previously will be lost as they're not saved and printed.

Maybe instead of returning the error in tree.go, we do this:
```go
level.Error(logger)...
return nil
```

Reply:
I've updated the code, PTAL